rackcity/models/asset_change_planner.py

```diff
@@ -44,12 +44,14 @@ class AssetCP(models.Model):
     rack_position = models.PositiveIntegerField()
     model = models.ForeignKey(
         ITModel,
-        on_delete=models.CASCADE,
+        on_delete=models.SET_NULL,
+        null=True,
         verbose_name="related model",
     )
```

Comment:
expected behavior if you the model or rack in the real world is deleted: the change planner will show a conflict saying "associated model was deleted"... this would also result in invalid network ports/power port connections, not sure how we want to handle this exactly, but will discuss with @juliasaveliff